### PR TITLE
added registerProtostreamType function

### PR DESCRIPTION
**Pull request by NeerajGartia21**
This is a PR for the issue #72 .

### CODE REVIEW

File: lib/protocols.js [hunk 1]
```diff
@@ -750,6 +750,7 @@
       return f.dispatch(
         f.isa('application/json', encoder(2, encoderJson, decoderJson()))
         , f.isa('text/plain', encoder(13, encoderString, decoderString()))
+        , f.isa('application/x-protostream', encoder(13, encoderProtobuf, decoderProtobuf()))
```
Comment (rigazilla) [inline lib/protocols.js:753]:
```suggestion
        , f.isa('application/x-protostream', encoder(12, encoderProtobuf, decoderProtobuf()))
```
values for media types are defined [here](Correct value for this is in https://github.com/infinispan/infinispan/blob/10220f7910942e2d2190c3933bd985ac0f6c493e/commons/all/src/main/java/org/infinispan/commons/dataconversion/MediaTypeIds.java#L44)

**Comment by rigazilla** [comment]
@NeerajGartia21 this looks good to me. If all the code for the _Setup Time_ (as describe [here](https://github.com/infinispan/js-client/issues/72#issuecomment-1187713036)) is in I would merge this and go ahead with the _Working Time_ code in a new PR, so to keep PRs small.
Wdyt?

**Comment by NeerajGartia21** [comment]
I think after committing requested changes all code for setup time are in.

**Comment by rigazilla** [comment]
merged, thanks @NeerajGartia21 !

**Pull request merged**